chart/flux/README.md

```diff
@@ -248,6 +248,8 @@ The following tables lists the configurable parameters of the Flux chart and the
 | `registry.dockercfg.enabled`                      | `false`                                              | Mount `config.json` via Secret into the Flux Pod, enabling Flux to use a custom docker config file
 | `registry.dockercfg.secretName`                   | `None`                                               | Kubernetes secret with the docker config.json
 | `registry.dockercfg.configFileName`               | `/dockercfg/config.json`                             | Alternative path/name of the docker config.json
+| `memcached.enabled`                               | `true`                                               | Create a memcached deployment and service. Set to `false` to use an external memcached service
```

Comment:
Please change this to `When set to false you must set an external memcached service`